### PR TITLE
[13.0][MIG] product_pack

### PR DESCRIPTION
**Pull request by jjscarafia**

### CODE REVIEW

**Comment by jjscarafia** [comment]
@pedrobaeza @ernestotejeda I can't get it right with format.
`product_pack/__manifest__.py:19:80: E231 missing whitespace after`
I'm loosing my head with this... :(

**Comment by jjscarafia** [comment]
Done @pedrobaeza @ernestotejeda 
Thanks

File: product_pack/models/product_product.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def price_compute(self, price_type, uom=False, currency=False, company=False):
                     pack_price = pricelist.currency_id._convert(
                         pack_price,
                         product.currency_id,
-                        self.company_id or self.env.user.company_id,
+                        self.company_id or self.self.env.company,
```
Comment (ernestotejeda) [inline product_pack/models/product_product.py]:
```suggestion
                        self.company_id or self.env.company,
```

**Review by ernestotejeda — COMMENTED** [review]
Thanks.
Functionally OK.
Code OK.

Just a suggested change.

**Comment by jjscarafia** [comment]
fixed @ernestotejeda 
thanks!

**Comment by pedrobaeza** [comment]
Please squash migration commits together.

**Comment by jjscarafia** [comment]
done @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**